src/pymatgen/electronic_structure/boltztrap.py

```diff
@@ -1468,7 +1468,7 @@ def is_isotropic(x, isotropy_tolerance) -> bool:
             return bool(
                 all([st[0], st[1], st[2]])
                 and (abs((st[1] - st[0]) / st[1]) <= isotropy_tolerance)
-                and (abs(st[2] - st[0]) / st[2] <= isotropy_tolerance)
```

Comment:
Seemingly wrong `()` position (compare with its siblings), should be the `abs` of difference

Reply:
revert in favor of a separate issue